packages/shell-api/src/helpers.ts

```diff
@@ -327,6 +327,14 @@ export async function getPrintableShardStatus(
           autosplit === null || autosplit.enabled ? 'yes' : 'no',
       };
     })(),
+    (async (): Promise<void> => {
+      // Is automerge currently enabled, available since >= 7.0
```

Comment:
So this is available since 7.0 which means in older versions it'd also similarly say `'yes'` which is weird.
The solution to this is to check for the server version and compare with semver but I don't see this being used anywhere
1. Do we want to introduce this kind of differentiation and hide this field < 7.0?
2. What does a server version mean for automerge in sharded cluster? If I understand right it's possible for shards to have i.e. mix of 6.0 and 7.0 Mongo versions, does one get it from i.e. adminDB or something of sort?

Reply:
Just wondering – what does `automerge === null` correspond to? Is that the case in which we would want to not print this information?

Reply:
Yeah just confirmed this more or less with `start/`stop` `autoMerger` is when that'd appear.

We could omit it as information which isn't explicitly set and show only if it has been configured; users looking for it would probably end up starting/stopping so seems reasonable.